DSCResources/MSFT_xHostsFile/MSFT_xHostsFile.psm1

```diff
@@ -0,0 +1,312 @@
+$Script:hostsFilePath = "${env:windir}\system32\drivers\etc\hosts"
+
+Import-LocalizedData -BindingVariable LocalizedData -filename MSFT_xHostsFile.psd1 -BaseDirectory $PSScriptRoot -Verbose
+
+function Get-TargetResource
+{
+    [OutputType([Hashtable])]
+    param (
+        [parameter(Mandatory = $true)]
+        [string]
+        $HostName,
+        [parameter(Mandatory = $true)]
+        [string]
+        $IPAddress
+    )
+
+    $configuration =
+    @{
+        HostName = $HostName
+        IPAddress = $IPAddress
+    }
+
+    Write-Verbose $localizedData.checkingHostsFileEntry
```

Comment:
I hate to be a pain, but are we able to eliminate the use of any positional parameters? In general it is recommended against for production code (https://blogs.technet.microsoft.com/heyscriptingguy/2012/04/22/the-problem-with-powershell-positional-parameters/).

e.g. Change
```powershell
Write-Verbose $localizedData.checkingHostsFileEntry
```
to:
```powershell
Write-Verbose -Message $localizedData.checkingHostsFileEntry
```